translations/de-DE.json

```diff
@@ -0,0 +1,267 @@
+    "dashboard.embedUrlParamExtension.include": "Einschließen",
+    "dashboard.embedUrlParamExtension.query": "Abfrage",
+    "dashboard.embedUrlParamExtension.timeFilter": "Zeitfilter",
+    "dashboard.embedUrlParamExtension.topMenu": "Hauptmenü",
+    "dashboard.emptyDashboardAdditionalPrivilege": "Sie benötigen zusätzliche Rechte, um dieses Dashboard zu bearbeiten.",
+    "dashboard.emptyDashboardTitle": "Dieses Dashboard ist leer.",
+    "dashboard.factory.displayName": "Dashboard",
+    "dashboard.featureCatalogue.dashboardDescription": "Eine Sammlung von Visualisierungen und gespeicherten Suchanfragen anzeigen und teilen.",
+    "dashboard.featureCatalogue.dashboardSubtitle": "Daten in Dashboards analysieren.",
+    "dashboard.featureCatalogue.dashboardTitle": "Dashboard",
+    "dashboard.fillDashboardTitle": "Dieses Dashboard ist leer. Füllen wir es!",
+    "dashboard.helpMenu.appName": "Dashboards",
+    "dashboard.howToStartWorkingOnNewDashboardDescription1": "Klicken",
+    "dashboard.howToStartWorkingOnNewDashboardDescription2": "Sie in der Menüleiste oben, um mit dem Hinzufügen von Bereichen zu beginnen.",
+    "dashboard.howToStartWorkingOnNewDashboardEditLinkAriaLabel": "Dashboard bearbeiten",
+    "dashboard.howToStartWorkingOnNewDashboardEditLinkText": "Bearbeiten",
+    "dashboard.listing. savedObjectWarning": "Nach Titel kann nicht gefiltert werden",
+    "dashboard.listing.createButtonText": "Erstellen Sie",
+    "dashboard.listing.createNewDashboard.combineDataViewFromOpenSearchDashboardsAppDescription": "Sie können Datenansichten aus jeder OpenSearch-Dashboards-App in einem Dashboard kombinieren und alles an einem Ort sehen.",
+    "dashboard.listing.createNewDashboard.createButtonLabel": "Neues Dashboard erstellen",
+    "dashboard.listing.createNewDashboard.newToOpenSearchDashboardsDescription": "Neu bei OpenSearch Dashboards? Öffnen Sie {SampleDataInstallLink} für einen Test.",
+    "dashboard.listing.createNewDashboard.sampleDataInstallLinkText": "Installieren Sie einige Beispieldaten",
+    "dashboard.listing.createNewDashboard.title": "Erstellen Sie Ihr erstes Dashboard",
+    "dashboard.listing.dashboardsTitle": "Dashboards",
+    "dashboard.listing.noItemsMessage": "Sieht aus, als hätten Sie keine Dashboards.",
+    "dashboard.listing.table.columnUpdatedAtDescription": "Letzte Aktualisierung des gespeicherten Objekts",
+    "dashboard.listing.table.columnUpdatedAtName": "Letzte Aktualisierung",
+    "dashboard.listing.table.descriptionColumnName": "Beschreibung",
+    "dashboard.listing.table.entityName": "Dashboard",
+    "dashboard.listing.table.entityNamePlural": "Dashboards,",
+    "dashboard.listing.table.titleColumnName": "Position",
+    "dashboard.listing.table.typeColumnName": "Typ",
+    "dashboard.panel.AddToLibrary": "Zur Bibliothek hinzufügen",
+    "dashboard.panel.clonedToast": "Geklonter Bereich",
+    "dashboard.panel.clonePanel": "Bereich Klonen",
```

Comment:
In all the panel texts, "Tafel" is better than "Bereich" (similar for `.removePanel.replacePanel`, or "Bereichsdaten" -> "Tafelsdaten" in `unableToMigratePanelData`). "Bereich" just means area/sector, "Tafel" is appropriate for boards displaying charts or some types of control panels.